code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2785,6 +2822,29 @@
 	bitesize = 2
 	food_flags = FOOD_SWEET | FOOD_ANIMAL | FOOD_LACTOSE
 
+/obj/item/weapon/reagent_containers/food/snacks/sliceable/caramelcake
+	name = "caramel cake"
+	desc = "A cake with added caramel"
+	icon_state = "caramelcake"
+	slice_path = /obj/item/weapon/reagent_containers/food/snacks/chocolatecakeslice
```

Comment:
you forgot to change this to caramelcakeslice

Reply:
should be fixed now